src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -179,14 +179,11 @@
                         this.defaultMessage = "An error occurred fetching Git commits";
                     });
             },
-            getParameterValues(values) {
-                if (values) {
-                    this.parameterValues.forEach((param, key) => {
-                        if (values[key].name == param.name) {
-                            param.value = values[key].value
-                        }
-                    })
+            getParameterValues(values, valid) {
+                if (valid) {
+                    this.parameterValues = [...values]
                 }
+                this.disableRun = !valid
```

Comment:
This function might unilaterally enable the run button even though other parts of the form are invalid. But I think this is only a theoretical risk - at the moment it is only the parameters field than can be invalid (ignoring the button being disabled when a run is submitted).